buildroot-external/kernel/v6.1.y/hassos.config

```diff
@@ -196,3 +196,6 @@ CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ASCII=y
 CONFIG_NLS_ISO8859_1=y
 CONFIG_NLS_UTF8=y
+
+CONFIG_MPTCP=y
+CONFIG_MPTCP_IPV6=y
```

Comment:
It doesn't really matter all that much, but can you move it up a bit to the other network/IP related configs?

Reply:
Sure, good idea. I just put them after IPv6, and before Netfilter options, similar to what we have in the Linux kernel.